examples/nextjs-13/template/package.json

```diff
@@ -10,7 +10,7 @@
     "lint": "concurrently \"cd apps/host && npm run lint\" \"cd apps/remote && npm run lint\""
   },
   "dependencies": {
-    "@module-federation/dashboard-plugin": "^2.8.0-beta.7",
+    "@module-federation/dashboard-plugin": "^2.8.0-beta.6",

```

Comment:
why did this get bumped back down?

Reply:
I need to release a new version with a fix that removes the skipPost from FederationDashboardPlugin and keep it only for NextFederationPlugin

Reply:
It was causing an issue with the post dashboard data